The Crawling Terror

scnycc opened the pull request.

scnycc committed:
Empezar a añadir The Crawling Terror

scnycc committed:
Empezar a añadir The Crawling Terror

scnycc committed:
gh

scnycc committed:
Merge branch 'develop' of https://github.com/scnycc/audiowho-novelas into develop

scnycc reviewed (commented):
lol

Bigomby reviewed (changes requested):
No puedes modificar archivos de otra novela.

Bigomby reviewed (changes requested):
No puedes meter ficheros HTML dentro de la carpeta `caps`.

Bigomby commented on novelas/the-crawling-terror/docs/caps/Cap01.tex:
```diff
@@ -0,0 +1,240 @@
+<<<<<<< HEAD
```
Esto no es format latex válido, tienes marcas de resolución de conflictos de git.

Bigomby commented on novelas/the-crawling-terror/docs/caps/Cap02.tex:
```diff
@@ -0,0 +1,214 @@
+<<<<<<< HEAD
```
Más marcas de resolución de conflictos.

Bigomby reviewed (changes requested):
Varios cambios.

scnycc commented:
Aún así me convirtió la novela perfectamente

scnycc committed:
Arreglado fallos.
Doble espacios en capítulos.
Enviada modificaciones de capítulos 1 y 2.
El prólogo tenía llaves que no hacían falta

Bigomby reviewed (changes requested):
- El documento de dedicatoria está vacío, añade algo o elimínalo.
- En la carpeta `pre` hay dos ficheros que tienen el mismo índice (02), hay que corregir eso.

scnycc committed:
Arreglado de nuevo los pre

scnycc commented:
Arregaldo

scnycc committed:
El título del capítulo estaba mal
